torchrl/objectives/a2c.py

```diff
@@ -234,6 +240,7 @@ def __init__(
         functional: bool = True,
         actor: ProbabilisticTensorDictSequential = None,
         critic: ProbabilisticTensorDictSequential = None,
+        reduction: str = "mean",
```

Comment:
I would use `None` as a default, it can come in handy if some day we change the default
smth like `if reduction is None: reduction="mean"` later in the constructor.

Reply:
I am not sure if I follow. Default `mean` ensures BC and seems to me a a reasonable choice. Also Torch losses have `mean` as default, which I think is consistent.

Reply:
It will still be "mean". But not having "mean" is good practice because it allows us to detect whenever the user is passing or not the reduction. 
Might come in handy at some point: for instance, if we want to change the default for one loss someday, we will just have to check whether the reduction is None or not.

Reply:
and how do you define a no-reduction option then?

Reply:
`"none"` (not `None`)